priv/repo/migrations/20170803180536_add_notification_configuration.exs

```diff
@@ -0,0 +1,17 @@
+defmodule CoursePlanner.Repo.Migrations.AddNotificationConfiguration do
+  use Ecto.Migration
+
+  def up do
+    alter table(:users) do
+      add :notified, :date
+      add :notification_frequency_days, :integer, null: false, default: 1
```

Comment:
*frequency* is the number of times an event happens in a given time, for example 3 notifications/day
*period* is the amount of time between two events, for example 1 day/notification

this is what comes in my mind when I see this naming, there may be other definitions which I am not aware of...